.github/workflows/conda-build.yml

```
@@ -5,54 +5,38 @@ on: [push, pull_request]
         conda activate quantmsutils
         conda list
-      shell: bash -l {0}
-
-    - name: Test the installed package
-      run: |
-        conda activate quantmsutils
         quantmsutilsc --help
-        quantmsutilsc --version
-      shell: bash -l {0}
\ No newline at end of file
+        quantmsutilsc --version
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)